src/js/video.js

```diff
@@ -107,45 +107,20 @@ setup.autoSetupTimeout(1, videojs);
 videojs.VERSION = '__VERSION__';
 
 /**
- * Get the global options object
+ * The global options object. These are the settings that take effect
+ * if no overrides are specified when the player is created.
  *
- * @return {Object} The global options object
- * @mixes videojs
- * @method getGlobalOptions
- */
-videojs.getGlobalOptions = () => globalOptions;
-
-/**
- * For backward compatibility, expose global options.
- *
- * @deprecated
- * @memberOf videojs
- * @property {Object|Proxy} options
- */
-videojs.options = createDeprecationProxy(globalOptions, {
-  get: 'Access to videojs.options is deprecated; use videojs.getGlobalOptions instead',
-  set: 'Modification of videojs.options is deprecated; use videojs.setGlobalOptions instead'
-});
-
-/**
- * Set options that will apply to every player
  * ```js
- *     videojs.setGlobalOptions({
- *       autoplay: true
- *     });
+ *     videojs.options.autoplay = true
  *     // -> all players will autoplay by default
  * ```
- * NOTE: This will do a deep merge with the new options,
- * not overwrite the entire global options object.
  *
- * @return {Object} The updated global options object
- * @mixes videojs
- * @method setGlobalOptions
+ * @type {Object}
  */
-videojs.setGlobalOptions = function(newOptions) {
-  return mergeOptions(globalOptions, newOptions);
-};
+videojs.options = defaultOptions;
+
 
+const players_ = {};
```

Comment:
Players still get added to Player.players [when they're initialized](https://github.com/videojs/video.js/blob/cd3f96978fff5fab3c1962ead04da4ffde40ec9f/src/js/player.js#L198), so right now this would always be empty.